src/main.rs

```diff
@@ -12,7 +12,7 @@ use logic::OwnLogic;
 #[derive(Parser, Debug)]
 struct Args {
     /// The game server's host address.
-    #[clap(long, default_value = "localhost")]
+    #[clap(short, long, default_value = "localhost")]
```

Comment:
IIRC this was to prevent ambiguities with `-h`/`--help`. How does clap handle this?

Reply:
```-h``` for help is overwritten:
```
socha-client-2024 -h
error: a value is required for '--host <HOST>' but none was supplied

For more information, try '--help'. 
```

Reply:
Ah okay, thanks for clarifying!